docs/design_docs/datanode_recovery_design_0513_2021.md

```diff
@@ -121,6 +123,48 @@ message DDLBinlogMeta {
 
   **O4-1.** DataNode scales flowgraph 2 Day
 
+Q. flowgraph pool?
+
+**DM DD position should be the same**
+- Change `WatchDmChannelsRequest` proto.
+``` proto
+message Vchannel {
+  int64 collectionID = 1;
+  string vchannel_name = 2;
+  internal.MsgPosition start_position = 3;
+  internal.MsgPosition end_position = 4;
+}
+
+message WatchDmChannelsRequest {
+  common.MsgBase base = 1;
+  Vchannel ddl_channel = 2;
+  repeated Vchannel vchannels = 3;
+}
```

Comment:
I'm not sure if this's fit @sunby

Reply:
I think there's a mistake, ddl-channels' positions are different to each segment.  I'll fix it now.